pyproject.toml

```diff
@@ -21,19 +21,32 @@ GitPython = "3.1.18"
 SQLAlchemy = "^1.4.31"
 click = "8.0.4"
 tabulate = "0.8.9"
+Sphinx = {version = "4.5.0", optional = true}
+sphinx-markdown-builder = {version = "0.5.5", optional = true}
```

Comment:
This dependency is used ?

Reply:
It's used in markdown generation command, I removed it in the latest commit